docs/swagger.yaml

```diff
@@ -5,6 +5,237 @@ definitions:
         additionalProperties: true
         type: object
     type: object
+  domain.Action:
```

Comment:
Can you generate the swagger before you push a new commit as it might be different because of new additions to the service